paymentsheet/src/main/java/com/stripe/android/paymentsheet/IntentConfirmationInterceptor.kt

```diff
@@ -177,8 +177,12 @@ internal class DefaultIntentConfirmationInterceptor @Inject constructor(
         shippingValues: ConfirmPaymentIntentParams.Shipping?,
         setupForFutureUsage: ConfirmPaymentIntentParams.SetupFutureUsage?,
     ): NextStep {
+        var attribution = paymentMethodCreateParams.attribution + "deferred-intent"
+        if (intentConfiguration.paymentMethodTypes.isNullOrEmpty()) {
+            attribution += "autopm"
```

Comment:
I don't see a test where we aren't adding this. Can we add one? I'd be happy to pair on it if you'd like.